### ISSUE TITLE
Does the GTK 4 work and will you keep updating it ?

### ISSUE THREAD

**Issue by Deottive** [TheGreatMcPain/gruvbox-material-gtk#15]
Just a casual question will you keep updating the theme for GTk 4 and further variats if so please do I literally love this theme

**Comment by TheGreatMcPain** [comment]
If [oomox](https://github.com/themix-project/oomox) already supports GTK4 I can re-generate the theme to support GTK4.

**Comment by BachoSeven** [comment]
@TheGreatMcPain I doubt they will be able to support it soon https://github.com/themix-project/oomox/issues/427

**Comment by Deottive** [comment]
Whenever they do please update the theme to GTK 4 because I have seen some themes on the internet which work for GTK 4 and whats better than GRUVBOX

**Comment by pietryszak** [comment]
I have no idea what about you talking about. This theme works with gtk apps. Even in folders in folder called gtk-4.0.
I install gnome-todo app created with gtk4.0 and a theme applay to it.

So...

![image](https://user-images.githubusercontent.com/45850093/195950546-f0e1ce92-a619-48cd-97c6-dae2de1e85b1.png)

Gnome-todo app gtk4
![image](https://user-images.githubusercontent.com/45850093/195988203-b995794a-12ef-4175-820c-ab99668cdb18.png)

**Comment by BachoSeven** [comment]
> @TheGreatMcPain I doubt they will be able to support it soon [themix-project/themix-gui#427](https://github.com/themix-project/themix-gui/issues/427)

It looks like they support it now https://github.com/themix-project/themix-gui/issues/427#issuecomment-1293243200 (?)

**Comment by TheGreatMcPain** [comment]
I've regenerated the theme with the latest version of themix-gui.  Does GTK-4 work?

**Comment by 8FordPrefect8** [comment]
No. I read somewhere else the recommendation to copy the content of gtk-4.0 to ~/.config/gtk-4.0. That has an effect. Eg. in nautilus surroundings of buttons and left folder bar is according to theme. But top bar and background is really bright according to gnome-settings.

**Comment by Sohail-XD** [comment]
Any update on gtk 4?
